tests/terraform/checks/resource/azure/test_AzureFrontDoorEnablesWAF.py

```diff
@@ -1,116 +1,42 @@
+import os
 import unittest
 
-import hcl2
-
+from checkov.runner_filter import RunnerFilter
+from checkov.terraform.runner import Runner
 from checkov.terraform.checks.resource.azure.AzureFrontDoorEnablesWAF import check
-from checkov.common.models.enums import CheckResult
 
 
-class TestAzureFrontDoorEnablesWAF(unittest.TestCase):
+class TestOSSBucketAccessLogs(unittest.TestCase):
```

Comment:
```suggestion
class TestAzureFrontDoorEnablesWAF(unittest.TestCase):
```
just adjust the test name and then it is good to merge 😉